Adding report page link to map (resolves #164)

lesliealanis opened the pull request:
Adding link to the map so that it goes to the report page

lesliealanis committed:
Adding report page link to map (resolves #164)

Adding link to the map so that it goes to the report page

Frijol commented on content/home/home-content.md:
```diff
@@ -4,7 +4,7 @@
 EPA is overseen by two congressional committees, the House Energy and Commerce Committee (E&C) and the Senate Environmental and Public Works Committee (EPW). EDGI took a look at environmental enforcement in committee members' home districts. Here's where those reps come from:
 
-![Map of representatives on EPA oversight committees, covered by EEW Congressional Report Cards project](./eew-map.png)
+[![Map of representatives on EPA oversight committees, covered by EEW Congressional Report Cards project](eew-map.png)](https://edgi-govdata-archiving.github.io/EEW-Website/reports "Map of representatives on EPA oversight committees, covered by EEW Congressional Report Cards project")
```
The label for the link should actually describe the target of the link (i.e. "Reports page of this website")

Frijol replied:
Link target should be `/reports`, which is a relative link (which will be important when we change from staging site to our real URL). We'll hit the error outlined in https://github.com/edgi-govdata-archiving/EEW-Website/issues/163 but I will put a note on that issue to check this out at the same time (when we go live)

lesliealanis committed:
adding kelsey's changes

Adding Kelsey's changes:
- changing the map link to the reports page so that it's a relative link
- also changing the link label to "EEW's Reports Page"

Frijol reviewed (approved):
Looks great, thanks!
